packages/auth-manager/openapi3.yaml

```diff
@@ -0,0 +1,839 @@
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: createdAfter
+          description: filters all clients created after given date
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: updatedBefore
+          description: filters all clients updated before given date
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: updatedAfter
+          description: filters all clients updated after given date
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: tags
+          description: filters based on tags
+          schema:
+            type: array
+            uniqueItems: true
+            items:
+              type: string
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/client'
+        400:
+          $ref: '#/components/responses/400BadRequest'
+    post:
+      operationId: createClient
+      summary: creates a new client
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/client'
+      responses:
+        201:
+          $ref: '#/components/responses/201Created'
+
+  /client/{clientName}:
+    parameters:
+      - $ref: '#/components/parameters/clientParam'
+    get:
+      operationId: getClient
+      summary: get client by name
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/client'
+        400:
+          $ref: '#/components/responses/400BadRequest'
+        404:
+          $ref: '#/components/responses/404NotFound'
+    patch:
+      operationId: updateClient
+      summary: update the client
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/namelessClient'
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/client'
+        400:
+          $ref: '#/components/responses/400BadRequest'
+        404:
+          $ref: '#/components/responses/404NotFound'
+  /client/{clientName}/connection:
+    parameters:
+      - $ref: '#/components/parameters/clientParam'
+    get:
+      operationId: getClientConnections
+      summary: gets the connections for a specific client
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/connection'
```

Comment:
createdAt is not required here, is it possible ?

Reply:
it is readonly so it doesn't matter.